ocs_ci/ocs/ui/views.py

```
@@ -1867,6 +1867,42 @@
+    # Notification drawer (bell icon in masthead) — alerts moved here in PF v6.
+    "notification-drawer-toggle": (
+        "//button[@data-quickstart-id='qs-masthead-notifications']",
+        By.XPATH,
+    ),
+    # Override pending-cleanup-alert-drpc-message to anchor the span search
+    # inside the notification drawer's Critical Alerts section.  Same text
+    # pattern as the pre-4.22 locator, just scoped to the drawer so it doesn't
+    # match stale DOM from a previously rendered page.
+    "pending-cleanup-alert-drpc-message": (
+        "//section[.//h2[normalize-space()='Critical Alerts']]"
+        "//li//span[contains(text(), \"DRPC '{}' in namespace '{}' requires manual cleanup\")]",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)